src/components/Header/HeaderContact.js

```diff
@@ -49,14 +67,7 @@ const HeaderContact = ({
 export default HeaderContact;
 
 HeaderContact.propTypes = {
-  title: PropTypes.string,
-  subTitle: PropTypes.string,
-  onPress: PropTypes.func,
-  titleStyle: PropTypes.object,
-  subtitleStyle: PropTypes.object,
-  containerStyle: PropTypes.object,
-  onPressSub: PropTypes.func,
-  disabledNextBtn: PropTypes.bool
+  onPressSub: PropTypes.func
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The propTypes for the `HeaderContact` component have been significantly reduced. This could potentially lead to less type checking and therefore more runtime errors. If these prop types are no longer necessary, this change is fine. However, if they were removed in error, they should be added back.
```diff
-  title: PropTypes.string,
-  subTitle: PropTypes.string,
-  onPress: PropTypes.func,
-  titleStyle: PropTypes.object,
-  subtitleStyle: PropTypes.object,
-  containerStyle: PropTypes.object,
-  onPressSub: PropTypes.func,
-  disabledNextBtn: PropTypes.bool
+  onPressSub: PropTypes.func
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->